JAMROZ-week 2 homework/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Don't forget to close your opening `<html>` tag with a `</html>` at the end of the document